docs/apidocs/automation_v2-secret-management.json

```diff
@@ -694,6 +694,13 @@
         "version" : {
           "type" : "integer",
           "format" : "int64"
+        },
+        "contentCreatedAtSeconds" : {
```

Comment:
It was generated as a side effect when I built Keywhiz to make sure the dependency removal didn't break anything; I could pull it into a separate PR or commit.